### PR TITLE
Feature/evidence note submit preference

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**

### CODE REVIEW

File: src/EA.Weee.Web/Scripts/weee-application.js [hunk 1]
```diff
@@ -201,8 +196,12 @@
             var autoCompletes = $(element.parentNode).find(".autocomplete__input");
             autoCompletes[0].classList.add("autocomplete__error");
         }
+        console.log(document.getElementById(element.id));
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Scripts/weee-application.js:199]: is that for local debugging only ?

File: src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml [hunk 2]
```diff
@@ -154,9 +154,10 @@
         <div class="govuk-!-margin-top-3">
 
             <div class="govuk-button-group">
+                @* This is a hidden button to force the default action to be save rather than submit*@
+                <button aria-hidden="true" class="govuk-visually-hidden" type="submit" name="Action" value="@ActionEnum.Save" tabindex="-1">Save</button>
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml:158]: do you need to add  **data-prevent-double-click="true"** as well ?

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml (hunk 2 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml:158]: No this button isnt clickable...

File: src/EA.Weee.Web/Scripts/weee-application.js (hunk 1 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Scripts/weee-application.js:199]: good spot...

File: src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml (hunk 2 above)
Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml:158]: ok

**Review by meglenapnovoselska — APPROVED** [review]

**Pull request merged**